### PR TITLE
Gracefully stop GRPC and update overwrites

### PR DESCRIPTION
**Pull request by jvdm**
## Checklist (N/A)

- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

## Testing Performed

go mod run cmd/scanner

### CODE REVIEW

**Review by daynewlee — APPROVED** [review]
LGTM

File: scanner/go.mod [hunk 1]
```diff
@@ -134,6 +134,7 @@ require (
 )
 
 replace (
+	github.com/stackrox/rox => ../
 	// The following is a manual copy of replacements done in github.com/stackrox/rox.
```
Comment (RTann) [inline scanner/go.mod:138]: I do wonder how necessary these replacements are

File: scanner/go.mod [hunk 2]
```diff
@@ -134,6 +134,7 @@ require (
 )
 
 replace (
+	github.com/stackrox/rox => ../
```
Comment (RTann) [inline scanner/go.mod:137]: This is fine for this PR, but I'm noticing that every change made to `github.com/stackrox/rox` will require a `go mod tidy` both in the main module and this module, which may be annoying for devs.

I'm wondering if we should just choose the current `master` branch and then let dependabot update this weekly for us

**Review by RTann — APPROVED** [review]

File: scanner/go.mod (hunk 1 above)
Comment (jvdm) [reply scanner/go.mod:138]: Go won't carry them over from `../`, so we must keep them up-to-date to ensure we pull the expected dependencies. Unless you know a way to verify which depends are actually being used by scanner through stackrox?

Comment (RTann) [reply scanner/go.mod:138]: yeah I can't think of anything. Guess we are stuck with this annoyance.

I was originally hoping we would not have to add the rox dependency, but doesn't seem like we can avoid that :(

**Pull request merged**